### DISCUSSION TITLE
Unable to Install XAML Hot Reload Preview 1

### DISCUSSION THREAD

**Discussion by TheRobo-13** [unoplatform/uno#8160] [Q&A]
when I open the Hot Reload installer:
![image](https://user-images.githubusercontent.com/91838037/155000619-db7e3e81-c5d9-43cb-aae2-95ebaca4b384.png)

In the install log:
`VSIXInstaller.NoApplicableSKUsException: This extension is not installable on any currently installed products.`

**Comment by jeromelaban** [comment]
Make sure to follow these steps: https://platform.uno/docs/articles/get-started-vs-2022.html

**Reply by TheRobo-13** [reply]
I did, the solution templates extention is downloaded

**Reply by jeromelaban** [reply]
Can you explain what you mean by the extension is downloaded? Please provide step by step information of what you did.
